kyo-prelude/shared/src/main/scala/kyo2/Ansi.scala

```diff
@@ -0,0 +1,111 @@
+package kyo2
+
+object Ansi:
+
+    extension (str: String)
+        def black: String   = s"\u001b[30m$str\u001b[0m"
```

Comment:
Is there any notable performance difference with string concatenation vs interpolation?